gems/smithy/lib/smithy/vise/structure_shape.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Vise
+    # Represents a structure shape from the model.
+    class StructureShape < Shape
+      # (see Shape#initialize)
+      def initialize(id, shape)
+        super
+        @members = shape['members']
```

Comment:
Members could have their own traits. Could we do use `Vise::Trait.new` for those as well? I think it would be great to have an easy way to access members within a `Vise::Structure` too.

Reply:
Yes. The example model did not have those, but we can add them here.